pkg/build/node-shim.js

```diff
@@ -1,6 +1,10 @@
 // This file includes a shim that will execute your task code.
 import airplane from "airplane";
+{{if .IsCodeDefinedTask}}
+import { {{.EntrypointFunc}} } from "{{.Entrypoint}}";
```

Comment:
You can rename the import here, so you don't need to change the code below. It also avoids any issues with the entrypoint's name clashing with other variable definitions in this file.

```suggestion
import { {{.EntrypointFunc}} as task } from "{{.Entrypoint}}";
```